src/accelerate/state.py

```diff
@@ -900,7 +900,11 @@ def __init__(
                         os.environ["XLA_DOWNCAST_BF16"] = str(0)
                         self.downcast_bfloat = False
             elif os.environ.get("ACCELERATE_USE_DEEPSPEED", "false") == "true" and not cpu:
-                self.deepspeed_plugin = deepspeed_plugin
+                # Just incase a user manually creates an `AcceleratorState`/bypasses the `Accelerator`
+                if not isinstance(deepspeed_plugin, (list, tuple)):
```

Comment:
It's not intended for users to later add another DS plugin, right? I.e. they need to pass all they'll ever need directly during `__init__`. I'm asking because otherwise, a tuple would not allow appending new plugins later.

Reply:
Correct